scripts/pre-commit-quality-checks.sh

```
@@ -85,6 +85,18 @@ staged_docs_validation_paths() {
   done < <(staged_files)
 }
 
+staged_prompt_validation_paths() {
+  local line
+  while IFS= read -r line; do
+    [ -z "${line}" ] && continue
+    case "${line}" in
+      packages/*/resources/prompts/*.md|packages/*/resources/prompts/**/*.md|packages/*/src/**/commands.py|scripts/check-prompt-commands.py|tests/unit/test_check_prompt_commands_script.py)
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 93.

(SC2221)